release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -147,6 +147,25 @@ The CNI VRF plug-in was previously introduced as a Technology Preview feature in
 
 For more information, see xref:../networking/multiple_networks/assigning-a-secondary-network-to-a-vrf.adoc#cnf-assigning-a-secondary-network-to-a-vrf[Assigning a secondary network to a VRF].
 
+[id="ocp-4-9-nw-timeout-configuaration-parameters"]
+==== Ingress controller timeout configuration parameters
+
+This release introduces four timeout configurations for the Ingress Controller `tuningOptions` parameter:
+
+* `clientTimeout` specifies how long a connection will be held open while waiting for a client response.
+
+* `serverFinTimeout` specifies how long a connection will be held open while waiting for the server response to the client that is closing the connection.
+
+* `serverTimeout` specifies how long a connection will be held open while waiting for a server/backend response.
+
+* `clientFinTimeout` specifies how long a connection will be held open while waiting for the client response to the server/backend closing the connection.
+
+* `tlsInspectDelay` specifies how long the router can hold data to find a matching route.
+
+* `tunnelTimeout` specifies how long a tunnel connection, including websockets, remains open while the tunnel is idle.
+
+For more information, see xref:../networking/ingress-operator.adoc#nw-ingress-controller-configuration-parameters[Ingress controller configuration parameters].
```

Comment:
When I click this link, it resolves at the top of the page instead of at that section. Is that because the {context} (`_configuring-ingress`) is missing from the end?